python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -974,8 +957,26 @@ def updateMonitoringInformation(jobs_):
         #logger.debug('Monitor jobs    : ' + repr([j.fqid for j in monitor_jobs]))
         #logger.debug('Requeue jobs    : ' + repr([j.fqid for j in requeue_jobs]))
 
+        # Split all the monitorable jobs into groups based on the
+        # credential used to communicate with DIRAC
```

Comment:
I was unsure where to put it at first but I think I will move it to `IBackend`. *TODO*